stand/defs.mk

```diff
@@ -128,6 +128,11 @@ SSP_CFLAGS=
 CFLAGS+=	-ffreestanding ${CFLAGS_NO_SIMD}
 .if ${MACHINE_CPUARCH} == "aarch64"
 CFLAGS+=	-mgeneral-regs-only -ffixed-x18 -fPIC
+# XXX-Morello: Disable Morello support in the bootloader. EFI needs to
+# understand capabilities before we can enable this as it may have interrupts
+# enabled so will need to save/restore the full capability registers.
+CFLAGS:=	${CFLAGS:N-march=morello*:N-mabi=purecap:N-femulated-tls}
```

Comment:
Hm, having looked at making RISC-V do EFI which requires no build system changes due to already providing an explicit -march/-mabi combination I feel like Morello should follow that model and specify `-march=morello+noa64c -mabi=sysv -fno-emulated-tls` as appropriate.